src/index.ts

```diff
@@ -77,4 +85,35 @@ export default class BlueButton {
           : SANDBOX_BASE_URL,
     };
   }
+
+  public getAccessTokenUrl(): string {
+    return this.baseUrl + "/" + this.version + "/o/token/";
+  }
+
+  public getAuthorizationUrl(): string {
+    return `${this.baseUrl}/${this.version}/o/authorize`;
```

Comment:
```suggestion
    return `${this.baseUrl}/v${this.version}/o/authorize`;
```
Version is just the number, without the v in front.

Reply:
Oops. Added the "v" in for this and other areas. TY!